### PR TITLE
CB-24869: Set up STIG_ENABLED flag across our automation

### PR DESCRIPTION
**Pull request by sgabor001**

### CODE REVIEW

File: packer.json [hunk 1]
```diff
@@ -120,7 +120,9 @@
     "image_uuid": "{{ env `IMAGE_UUID` }}",
     "cloud_provider": "{{ env `CLOUD_PROVIDER` }}",
     "ssh_public_key": "{{ env `SSH_PUBLIC_KEY` }}",
-    "fips_mode": "{{ env `FIPS_MODE` }}"
+    "fips_mode": "{{ env `FIPS_MODE` }}",
+    "stig_enabled": "{{ env `STIG_ENABLED` }}"
```
Comment (Bajzathd) [inline packer.json:124]: This is not needed

File: scripts/generate-tags.sh [hunk 2]
```diff
@@ -18,6 +18,10 @@ if [ -f /var/log/hardening ]; then
     TAGS=$(echo $TAGS | jq -r --arg hardening "$HARDENING" '. + {"hardening": $hardening}')
 fi
 
+if [[ "$STIG_ENABLED" == true ]]; then
```
Comment (Bajzathd) [inline scripts/generate-tags.sh:21]: This will be handled by the previous hardening file

File: saltstack/final/salt/stig/init.sls
Comment (Bajzathd) [inline saltstack/final/salt/stig/init.sls:1]: I don't see any includes of this file, how will this run?

File: packer.json (hunk 1 above)
Comment (sgabor001) [reply packer.json]: deleted

File: saltstack/final/salt/stig/init.sls
Comment (sgabor001) [reply saltstack/final/salt/stig/init.sls:1]: obsolete

File: scripts/generate-tags.sh (hunk 2 above)
Comment (sgabor001) [reply scripts/generate-tags.sh]: obsolete

**Review by Bajzathd — APPROVED** [review]

**Pull request merged**